operators/bosh-operator/DirectorService.js

```diff
@@ -921,11 +922,15 @@ class DirectorService extends BaseDirectorService {
     const directorService = new DirectorService(plan, instanceId);
     return Promise
       .try(() => context ? context : directorService.platformContext)
-      .then(context => directorService.assignPlatformManager(DirectorService.getPlatformManager(context.platform)))
+      .then(context => directorService.assignPlatformManager(DirectorService.getPlatformManager(context)))
       .return(directorService);
   }
 
-  static getPlatformManager(platform) {
+  static getPlatformManager(context) {
+    let platform = context.platform;
+    if (platform === CONST.PLATFORM.SM) {
+      platform = context.origin;
+    }
     const PlatformManager = (platform && CONST.PLATFORM_MANAGER[platform]) ? require(`../../broker/lib/fabrik/${CONST.PLATFORM_MANAGER[platform]}`) : ((platform && CONST.PLATFORM_MANAGER[CONST.PLATFORM_ALIAS_MAPPINGS[platform]]) ? require(`../../broker/lib/fabrik/${CONST.PLATFORM_MANAGER[CONST.PLATFORM_ALIAS_MAPPINGS[platform]]}`) : undefined);
     if (PlatformManager === undefined) {
       return new BasePlatformManager(platform);
```

Comment:
@amitmalav As discussed, [platformContext()](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/5179c09b2c1c2859bb752d8fb4309d83aaeb9197/operators/bosh-operator/DirectorService.js#L58) will be changed as @jayeshjk separate PR.